.travis.yml

```diff
@@ -272,7 +272,8 @@ jobs:
     - if: (type = pull_request) OR (type = cron) OR NOT (branch = master)
       stage: test
       name: "Regression 9.6.17"
-      env: PG_VERSION=9.6.17
+      env:
+        - PG_VERSION=9.6.17
```

Comment:
I don't follow why this change is needed. It makes it inconsistent with the jobs below.

Reply:
i'll remove i had multiple envs there for a while so this is leftover